synthtool/gcp/common.py

```diff
@@ -57,6 +57,7 @@ def node_library(self, **kwargs) -> Path:
         #  .repo-metadata.json, or excluding README.md, we can remove this.
         if not os.path.exists("./.repo-metadata.json"):
             self.excludes.append("README.md")
+            self.excludes.append("samples/README.md")
```

Comment:
we don't want to generate the new README until we've plopped a `.repo-metadata.json` in the folder, otherwise we get a junk file.